### PR TITLE
Linter task

### PR DESCRIPTION
**Pull request by ameenross**
At first I wanted to make a big PR with a couple of tasks, but I think I'm gonna keep it granular.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            "name": "Ameen Ross",
+            "email": "a.ross@amdev.eu"
+        }
+    ],
+    "minimum-stability": "dev",
+    "require": {
+        "jakub-onderka/php-parallel-lint": "0.*"
+    },
+    "require-dev": {
+        "bldr-io/bldr": "~7.0",
```
Comment (cryptiklemur) [inline composer.json:15]: Just updated to 8.0.0 the other day. as a heads up. This should still work on 7.0 though

File: src/Task/Output/Output.php [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+{
+    /**
+     * @var OutputInterface $realOutput
+     */
+    private $realOutput;
+
+    /**
+     * @param Linter\IWriter $writer
+     */
+    public function __construct(Linter\IWriter $writer)
```
Comment (cryptiklemur) [inline src/Task/Output/Output.php:30]: can we get rid of this? or is something else going in here?

**Comment by cryptiklemur** [comment]
Is this a WIP? or ready to go?

**Comment by ameenross** [comment]
The output is there because otherwise the library will print its output directly to console. This was the proper way of ensuring that it went to the injected symfony console output class.

It's working, apart from the fact that there is no readme I think this is good to go. Maybe it should be merged to dev branch until all tasks are there though?

File: src/Task/Output/Output.php (hunk 2 above)
Comment (ameenross) [inline src/Task/Output/Output.php:30]: You mean just the constructor? I overrode it because in Linter\TextOutput it does redundant stuff for our usecase.

Comment (cryptiklemur) [inline src/Task/Output/Output.php:30]: Ahh, alrighty. Makes sense. Should make a comment of that

**Comment by cryptiklemur** [comment]
Good call. I've made a develop branch. Just open a new PR

**Pull request closed without merging**